docs/development/decision-records/2024-09-16-proposal-federated-catalog-distribution/README.md

```diff
@@ -0,0 +1,33 @@
+# Proposal for FederatedCatalog with Tractus-X distribution
+
+## Decision
+
+The **proposed solution is to have a Federated Catalog as own runnable service** capable of crawling all the wanted catalogs and expose that data.
+
+
+## Rationale
+
+The Federated Catalog aims at providing the aggregation of catalogs from multiple participants in a centralized point to reduce latency.
+
+Choosing a solution that decouples from Control Plane (like the one used for the Data Plane) and able to be scalable will future-proof the Federated Catalog as a feature and embraces wider usage. 
+Like so, having a Federated Catalog service, scalable on its own based on user needs, and not on Control Plane, allows the best resource management in the long term. It also enables adding features to the Federated Catalog without the need to change the Control Plane logic (assuming no API usage change).
```

Comment:
This was bad phrasing on my side, I was aiming at mentioned the goal of having independent updates. However, in the previous paragraph was already mentioned the decoupling this approach promotes so it (like the scalability) can be seen as repetitive information.
Removed in [here](https://github.com/eclipse-tractusx/tractusx-edc/pull/1555/commits/a452459467e7d78d60ce263a2fecf6ea2c14c205).

Reply:
in any case also the "indipendent updates" is not something we want to achieve with this decision, because we will release the FC together with CP and DP, otherwise we would put it in a separate chart.

Reply:
I see. I wanted to highlight, as an example, that if a change in the Federated Catalog Cache or similar should be transparent for other instances (like CP or DP). And it would be, I believe, but updating to a new version when all are in tractus-edc charts then "independent update" is not accurate.
Thanks!